src/Chunk.cpp

```diff
@@ -974,7 +974,21 @@ cItems cChunk::PickupsFromBlock(Vector3i a_RelPos, const cEntity * a_Digger, con
 	GetBlockTypeMeta(a_RelPos, blockType, blockMeta);
 	auto blockHandler = cBlockInfo::GetHandler(blockType);
 	auto blockEntity = GetBlockEntityRel(a_RelPos);
-	auto pickups = blockHandler->ConvertToPickups(blockMeta, blockEntity, a_Digger, a_Tool);
+	cItems pickups (0);
+	bool canHarvestBlock;
+	if (a_Tool)
+	{
+		auto toolHandler = a_Tool->GetHandler();
+		canHarvestBlock = toolHandler->CanHarvestBlock(blockType);
+	}
+	else
+	{
+		canHarvestBlock = false;
```

Comment:
Some blocks can be harvested with an empty hand. The empty handler should cover this:
```cpp
auto toolHandler = a_Tool ? a_Tool->GetHandler() : cItemHandler::GetItemHandler(E_ITEM_EMPTY);
```

Reply:
Accounted for that